common/utils/tests/Test_GenomeUtils.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+* Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+*
+* NVIDIA CORPORATION and its licensors retain all intellectual property
+* and proprietary rights in and to this software, related documentation
+* and any modifications thereto.  Any use, reproduction, disclosure or
+* distribution of this software and related documentation without an express
+* license agreement from NVIDIA CORPORATION is strictly prohibited.
+*/
+
+#include <claragenomics/utils/genomeutils.hpp>
+
+#include "gtest/gtest.h"
+
+namespace claragenomics
+{
+
+namespace genomeutils
+{
+
+TEST(GenomeUtilsTest, ReverseComplement)
+{
+    std::string genome("ATCGAACGTATG");
+    char complement[genome.length() + 1];
```

Comment:
Runtime sized arrays on the stack are not valid C++, but an extension some compilers (e.g. GCC) provide. Just use an `std::string`.

Reply:
since the API takes in a `char *` to copy the reverse complement into, I converted this into a unique ptr